src/components/flatParameters/FlatParameters.tsx

```diff
@@ -424,7 +433,11 @@ export function FlatParameters({
         <List sx={styles.paramList}>
             {paramsAsArray.map((param, index) => (
                 <Fragment key={param.name}>
-                    <ListItem sx={styles.paramListItem}>
+                    <ListItemButton
+                        sx={styles.paramListItem}
+                        disabled={disabledParams !== undefined && disabledParams.indexOf(param.name) >= 0}
```

Comment:
```suggestion
                        disabled={disabledParams?.indexOf(param.name) >= 0}
```
Use JS syntax

Reply:
I don't understand why we have double checks, one as this level, and another in renderField?

Reply:
Yes the one in renderField() is to removed

Reply:
The switch is in the ListItemButton. It's enough to disable the ListItemButton

Reply:
ok solved